src/app/components/angucomplete/angucomplete.directive.js

```diff
@@ -0,0 +1,274 @@
+/**
```

Comment:
Do not copy paste external libraries that you are using. Use nmp/bower to add such dependencies and load them.